docs/notifications.md

```diff
@@ -85,7 +85,7 @@ Recognized stream names:
  * `notification_from_execution`
    Filter: `contract` field containing a string with hex-encoded Uint160 (LE
    representation) and/or `name` field containing a string with execution 
-   notification name.
+   notification name which should not be longer than 32 symbols.
```

Comment:
Symbol is not equal to byte, read the https://go.dev/blog/strings:
```
This technique is handy when much of the string is intelligible as text but there are peculiarities to root out; it produces:

"\xbd\xb2=\xbc ⌘"

If we squint at that, we can see that buried in the noise is one ASCII equals sign, along with a regular space, and at the end appears the well-known Swedish “Place of Interest” symbol. That symbol has Unicode value U+2318, encoded as UTF-8 by the bytes after the space (hex value 20): e2 8c 98.
```
Should be something like "which should be a valid UTF-8 string not longer than 32 bytes"

Reply:
fixed